ci/docker-content-tag.sh

```diff
@@ -67,7 +67,8 @@ EOF
   docker container run --rm "${iid}" |
     LC_ALL=C sort -k1.67 | # 67 is the offset of the filename after the hash
     sha256sum -b |
-    cut -c-5
+    cut -c-5 |
+    sed "s/$/-${versionTag}/"
```

Comment:
What happens when we don't include a versionTag, like for production? I think this would leave a trailing dash. Not a major issue but not terribly "pretty". I'm not intending to block the PR on that though. One solution might be to have a default value for versionTag if its not set.

Reply:
Check out this link for an example https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash

Reply:
I modified this to include the dash in argument rather than in the `sed` value.